### ISSUE TITLE
Feature Request: Hide "Live/Scheduled/Premieres"

### ISSUE THREAD

**Issue by JasonJAyalaP** [lawrencehook/remove-youtube-suggestions#59]
I hide, in my subscriptions page, the distracting  Live / Schedule / Premiers thumbnails.

I've been doing this with ublock custom filters:

```
youtube.com##ytd-grid-video-renderer.ytd-grid-renderer.style-scope:has(#metadata-line:has-text(/(Scheduled|Premieres)/i))
youtube.com##ytd-grid-video-renderer.ytd-grid-renderer.style-scope:has(#video-badges:has-text(/Live/i))
youtube.com##ytd-grid-video-renderer.ytd-grid-renderer.style-scope:has(.ytd-badge-supported-renderer.style-scope.badge-style-type-live-now-alternate.badge)

```
EDIT: Updated code to handle changes in "Live". Those last two lines are two ways to handle "Live" thumbnails (not sure which will be more futureproof).

**Comment by lawrencehook** [comment]
Hi @JasonJAyalaP 

Good idea! Thanks for making the issue. I'll see what I can do.

**Comment by lawrencehook** [comment]
@JasonJAyalaP Just implemented this for "Live". But having a harder time with scheduled and premieres. Do you happen to have any examples of channels with "Scheduled" and "Premieres" videos?

**Comment by lawrencehook** [comment]
@JasonJAyalaP Implemented as of 4.3.10. Thanks again for making the issue.

**Comment by lawrencehook** [comment]
![image](https://user-images.githubusercontent.com/6572030/197041298-4cd9efa9-4b9d-469b-8f86-b5f2af7cbfb9.png)
